# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qemu/qemu-0.9.1-enforce-16byte-stack-boundary.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

2008-01-12  Mike kronenberg  <mike.kronenberg@kronenberg.org>

	* Fix for QEMU 0.9.1.
	
2007-02-01  Mike kronenberg  <mike.kronenberg@kronenberg.org>

	* Fix for QEMU 0.9.0.
	
2006-02-12  Gwenole Beauchesne  <gbeauchesne@mandriva.com>

	* Enforce 16-byte boundaries.

--- qemu-0.8.0/Makefile.target.enforce-16byte-stack-boundary	2006-02-12 17:46:54.000000000 +0100
+++ qemu-0.8.0/Makefile.target	2006-02-12 17:55:24.000000000 +0100
@@ -67,7 +67,7 @@ endif
 
 ifeq ($(ARCH),i386)
 HELPER_CFLAGS+=-fomit-frame-pointer
-OP_CFLAGS+=-mpreferred-stack-boundary=2 -fomit-frame-pointer
+OP_CFLAGS+=-fomit-frame-pointer
 ifdef TARGET_GPROF
 USE_I386_LD=y
 endif
